common/utils.js

```diff
@@ -15,7 +15,19 @@ const {
 const { normalizeBech32 } = require('@cosmjs/encoding');
 
 function loadConfig(env) {
-    return require(`${__dirname}/../axelar-chains-config/info/${env}.json`);
+    const config = require(`${__dirname}/../axelar-chains-config/info/${env}.json`);
+
+    if (!config.sui) {
```

Comment:
we should do this generically for any chain, or remove this. I'd suggest just adding this example local.json to the readme instead of creating it dynamically